app/assets/stylesheets/vars/_spacing-units.scss

```diff
@@ -0,0 +1,36 @@
+$su-1: 0.25rem; // 4px
+$su-2: 0.5rem; // 8px
+$su-3: 0.75rem; // 12px
+$su-4: 1rem; // 16px
+$su-5: 1.25rem; // 20px
+$su-6: 1.5rem; // 24px
+$su-7: 2rem; // 32px
+$su-8: 3rem; // 48px
+$su-9: 4rem; // 64px
+$su-10: 8rem; // 128px
+
+$spacing-units: (
```

Comment:
I'm adding these two extra variables containing all spacing units, so it's easier to reference them in other [places](https://github.com/thepracticaldev/dev.to/pull/6334/files#diff-5f06803a04b2e9c226669b9fb6063bedR12).